### ISSUE TITLE
Ruby gem tries to load envkey-source on Rails

### ISSUE THREAD

**Issue by gordysc** [envkey/envkey#4]
Aloha! I'm following the README for the Ruby library for Ruby on Rails and getting the following error:

sh: /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/envkey-2.0.6/ext/envkey-source_2.0.22_darwin_arm64/envkey-source: No such file or directory

My understanding is we aren't "required" to have envkey-source installed? I also hit the same issue if I install envkey-source on my machine

Steps to Reproduce

1. Generate a .envkey in the Rails app root directory
2. Add the `envkey` gem to the Gemfile
3. Add `dotenv-rails` gem to the Gemfile (loads .env into ENV)
4. Add the ENVKEY server value to .env
5. Start the Rails server or Rails console using the mentioned command:

```sh
es -w --rolling -- rails s
# OR
es -w --rolling -- rails c
```

Stack Trace

```sh
Traceback (most recent call last):
	26: from bin/rails:4:in `<main>'
	25: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	24: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	23: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/commands.rb:18:in `<main>'
	22: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/command.rb:48:in `invoke'
	21: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/command/base.rb:87:in `perform'
	20: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor.rb:392:in `dispatch'
	19: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/invocation.rb:127:in `invoke_command'
	18: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/thor-1.2.1/lib/thor/command.rb:27:in `run'
	17: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/commands/console/console_command.rb:101:in `perform'
	16: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/command/actions.rb:14:in `require_application_and_environment!'
	15: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/railties-7.0.2.3/lib/rails/command/actions.rb:22:in `require_application!'
	14: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	13: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	12: from /Users/userx/Personal/walkster/config/application.rb:19:in `<main>'
	11: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.2.33/lib/bundler.rb:175:in `require'
	10: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.2.33/lib/bundler/runtime.rb:44:in `require'
	 9: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.2.33/lib/bundler/runtime.rb:44:in `each'
	 8: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.2.33/lib/bundler/runtime.rb:55:in `block in require'
	 7: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.2.33/lib/bundler/runtime.rb:55:in `each'
	 6: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bundler-2.2.33/lib/bundler/runtime.rb:60:in `block (2 levels) in require'
	 5: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	 4: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:30:in `require'
	 3: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/envkey-2.0.6/lib/envkey.rb:4:in `<main>'
	 2: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/envkey-2.0.6/lib/envkey/core.rb:16:in `load_env'
	 1: from /Users/userx/.rbenv/versions/2.7.4/lib/ruby/2.7.0/json/common.rb:156:in `parse'
/Users/userx/.rbenv/versions/2.7.4/lib/ruby/2.7.0/json/common.rb:156:in `parse': 783: unexpected token at 'sh: /Users/userx/.rbenv/versions/2.7.4/lib/ruby/gems/2.7.0/gems/envkey-2.0.6/ext/envkey-source_2.0.22_darwin_arm64/envkey-source: No such file or directory (JSON::ParserError)
```

**Comment by danenania** [comment]
envkey-source is bundled inside the Rails gem, but it looks like it's missing or there's a version mismatch. Fixing now!

**Comment by danenania** [comment]
Fixed in 2.0.8

**Comment by gordysc** [comment]
Thanks @danenania ! That worked!
